auth/src/main/java/com/alibaba/nacos/auth/exception/ConfigException.java

```diff
@@ -0,0 +1,37 @@
+package com.alibaba.nacos.auth.exception;
+
+/*
+ * Copyright 1999-2021 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.alibaba.nacos.api.exception.NacosException;
+
+/**
+ * AuthConfig exception.
+ *
+ * @author wuyfee
+ */
+public class ConfigException extends NacosException {
```

Comment:
Whether contains ambiguity?

How users can diff from `config` module and `auth` module?